src/server/node_manager.c

```diff
@@ -1096,7 +1096,7 @@ momptr_down(mominfo_t *pmom, char *why)
  *
  * @param[in]	op - The operation for the state change
  *
- * @return	char*
+ * @return	char *
  */
 char*
```

Comment:
Mind putting a space between the char and the *?  It'll make the code inline with our coding standards.

Reply:
Done.